vlib/v/ast/README.md

```diff
@@ -2,22 +2,24 @@
 
 ## Overview
 
-The Vlang abstract syntax tree is implemented by using sum type. 
+The V language abstract syntax tree uses sum types for its nodes.
 
-All the AST struct declarations can be found in V source code: [vlib/v/ast/ast.v](https://github.com/vlang/v/blob/master/vlib/v/ast/ast.v).
+All of the AST struct declarations, can be found in V's source code file:
+[vlib/v/ast/ast.v](https://github.com/vlang/v/blob/master/vlib/v/ast/ast.v).
 
 ## AST tool
 
-If you are new to the V's AST, you can install the [vast tool](https://github.com/lydiandy/vast). 
+If you are new to V's AST, you can install [the vast tool](https://github.com/lydiandy/vast). 
 
-It will generate example code to AST json file, which can help you understand the AST better.
+It will generate a json file representing the AST of a given .v file.
+That can help you with understanding the V AST better.
```

Comment:
"It will generate a JSON file representing the AST of a given .v file,
which can help you understand the AST better."

The document is about V's AST, so there's no reason to keep repeating that is it V's AST.

Also, JSON is an acronym, and acronyms should always be uppercase (that needs to be changed in a LOT of places).